### PR TITLE
Updated Amet Finance

### PR DESCRIPTION
**Pull request by amet-finance-core**
Updated:

- description
- audit links and count
- category
- chains

### CODE REVIEW

File: defi/src/protocols/data3.ts [hunk 1]
```diff
@@ -24511,21 +24511,25 @@ const data3: Protocol[] = [
     address: null,
     symbol: "-",
     url: "https://amet.finance",
-    description: "Amet Finance is a decentralized finance platform for on-chain bonds, providing secure financial transactions and investment opportunities in the blockchain ecosystem",
-    chain: "Manta",
+    description: "Amet Finance is a decentralized platform offering on-chain bond issuance and management services. It enables users to issue, trade, and manage bonds directly on the blockchain, providing a transparent, secure, and efficient environment for both issuers and investors. Amet Finance emphasizes user autonomy, allowing for customized bond terms and leveraging blockchain technology to democratize access to financial instruments.",
+    chain: "Base",
     logo: `${baseIconsUrl}/amet-finance.jpg`,
-    audits: "2",
+    audits: "4",
```
Comment (Define101) [inline defi/src/protocols/data3.ts:24517]: 2 instead of 4 please (the number)

File: defi/src/protocols/data3.ts [hunk 2]
```diff
@@ -24511,21 +24511,25 @@ const data3: Protocol[] = [
     address: null,
     symbol: "-",
     url: "https://amet.finance",
-    description: "Amet Finance is a decentralized finance platform for on-chain bonds, providing secure financial transactions and investment opportunities in the blockchain ecosystem",
-    chain: "Manta",
+    description: "Amet Finance is a decentralized platform offering on-chain bond issuance and management services. It enables users to issue, trade, and manage bonds directly on the blockchain, providing a transparent, secure, and efficient environment for both issuers and investors. Amet Finance emphasizes user autonomy, allowing for customized bond terms and leveraging blockchain technology to democratize access to financial instruments.",
+    chain: "Base",
```
Comment (Define101) [inline defi/src/protocols/data3.ts:24515]: keep this as Manta

File: defi/src/protocols/data3.ts [hunk 3]
```diff
@@ -24511,21 +24511,25 @@ const data3: Protocol[] = [
...
     logo: `${baseIconsUrl}/amet-finance.jpg`,
-    audits: "2",
+    audits: "4",
     audit_note: null,
     gecko_id: null,
     cmcId: null,
-    category: "Services",
-    chains: ["Manta"],
+    category: "Lending",
```
Comment (Define101) [inline defi/src/protocols/data3.ts:24521]: Why change the category?

Comment (amet-finance-core) [reply defi/src/protocols/data3.ts:24521]: At first we selected it wrong, but I switched back to Services, so you won't have UI problems

**Comment by Define101** [comment]
@amet-finance-core can we also remove changes to this file? .gitignore

**Comment by amet-finance-core** [comment]
> @amet-finance-core can we also remove changes to this file? .gitignore

What's the problem? It's .gitignone, for .vscode was there, so I added .idea for webstorm as well.

**Comment by Define101** [comment]
https://github.com/DefiLlama/defillama-server/pull/6631
Should be live within 2 hours

**Comment by realdealshaman** [comment]
closing since the changes were applied

**Pull request closed without merging**